packages/ui/Input.tsx

```diff
@@ -2,10 +2,9 @@ import clsx from 'clsx';
 import dynamic from 'next/dynamic';
```

Comment:
This too!! not installed, I'm sure if we had pnpm it could have yelled on us, yarn is doing some magic, can we also just try with pnpm to make sure everything is installed properly?

Reply:
@harish-sethuraman IMO we dont want to install next for this one import, we can use react lazy instead here

Reply:
We have image import too.